modules/schematics-core/utility/ngrx-utils.ts

```diff
@@ -90,11 +90,13 @@ export function addReducerToStateInterface(
     return new NoopChange();
   }
 
+  let state = stringUtils.camelize(options.name);
```

Comment:
```suggestion
  let state = options.plural ? stringUtils.pluralize(options.name) : stringUtils.camelize(options.name);
```

Let's use the pluralize method instead of tacking on an `s`.

Reply:
done